changelog/unreleased/dropzone-color

```diff
@@ -0,0 +1,5 @@
+Change: Use primary text color for the dropzone
```

Comment:
Enhancement since it's not a breaking change, right?

Reply:
Not breaking change but still a change 🤷  Label for enhancement from github: `New feature or request`

Reply:
`Feature` "Add readable text in dropzone" or `Bugfix` "Make text in dropzone readable when theming"

Reply:
Not really in for a mayor release on a one line CSS change ;)

Reply:
Oh true. I thought we already discussed it @LukasHirt . We wanted to prefix all changelog files with `change-`, `enhancement-` or `bugfix-` (so that the impact of the change becomes visible on a file level already) and that `Change` is used for breaking changes (= major release), `Enhancement` for non-breaking changes (= minor release) and `Bugfix` for bugfixes (= patch release).